packages/astro/src/content/utils.ts

```diff
@@ -186,6 +193,13 @@ function isOnIgnoreList(fileName: string) {
 	return ['.DS_Store'].includes(fileName);
 }
 
+/**
+ * Return if a file extension is a valid image asset, so we can avoid outputting a warning for them.
+ */
+function isImageAsset(fileExt: string) {
+	return [...VALID_INPUT_FORMATS, 'svg'].includes(fileExt.slice(1));
```

Comment:
nit: `[...VALID_INPUT_FORMATS, 'svg']` can be moved outside this function, defined as a `const`. This removes the need to create a "static" array every time.